Mods/ModVarietyPack/Defs/ThingDefs/Apparel_MVP.xml

```diff
@@ -2952,10 +2952,10 @@ And the surgical mask prevents unwanted bacteria entering the vic.. patient</des
 
 
 
-  <!--ArchNem© Ski™ goggles-->
+  <!--ArchNem� Ski� goggles-->
   <ThingDef ParentName="ApparelBase">
     <defName>Apparello_Brimsk</defName>
-    <label>ArchNem© Ski™ goggles</label>
+    <label>ArchNem� Ski� goggles</label>
```

Comment:
strange, as I have it set UTF-8, but if you look at the first line it is replacing it.